src/app/community/community.component.html

```diff
@@ -27,7 +27,7 @@ <h3>
         </mat-tab>
         <mat-tab i18n-label label="Services">
           <planet-markdown [content]="team.description || ''"></planet-markdown>
-          <button *planetAuthorizedRoles (click)="openDescriptionDialog()" mat-stroked-button i18n>Edit {
+          <button *planetAuthorizedRoles="planetCode !== user.code ? 'only' : ''" (click)="openDescriptionDialog()" mat-stroked-button i18n>Edit {
```

Comment:
Let's use an `ng-container` with `*ngIf` instead of changing the value of `planetAuthorizedRoles`.  It reads easier because this isn't an expected use for `planetAuthorizedRoles`.